machine1/dbus.go

```diff
@@ -163,7 +163,7 @@ func (c *Conn) DescribeMachine(name string) (machineProps map[string]interface{}
 	for key, val := range dbusProps {
 		machineProps[key] = val.Value()
 	}
-	return
+	return machineProps, nil
```

Comment:
Hmm, I always thought this to be more of a stylistic difference rather then functional? did this really fix the bug?

Reply:
well... I think so? maybe the original intention was to pass `machineProps` by reference (maps are always passed by reference in go IIRC), I will check on Monday removing this and see if the unit test below still pass :)

Reply:
alright, nvm you are right.

```
# pallotron @ devvm3927 in ~/src/go-systemd/machine1 on git:main x [9:22:08]
$ git diff
diff --git a/machine1/dbus.go b/machine1/dbus.go
index 49207ea..79c30ea 100644
--- a/machine1/dbus.go
+++ b/machine1/dbus.go
@@ -163,7 +163,7 @@ func (c *Conn) DescribeMachine(name string) (machineProps map[string]interface{}
        for key, val := range dbusProps {
                machineProps[key] = val.Value()
        }
-       return machineProps, nil
+       return
 }

 // KillMachine sends a signal to a machine

# pallotron @ devvm3927 in ~/src/go-systemd/machine1 on git:main x [9:22:24]
$ sudo go test -v
=== RUN   TestMachine
    dbus_test.go:132: machine machined-test-register-tshpleci properties: map[Class:container Id:[] Leader:4058823 Name:machined-test-register-tshpleci NetworkInterfaces:[] RootDirectory: Service:go-systemd State:running Timestamp:1729268546548624 TimestampMonotonic:269560782174 Unit:machined-test-register-tshpleci.service]
    dbus_test.go:132: machine machined-test-register-with-network-segaprxj properties: map[Class:container Id:[] Leader:4058825 Name:machined-test-register-with-network-segaprxj NetworkInterfaces:[] RootDirectory: Service:go-systemd State:running Timestamp:1729268546549606 TimestampMonotonic:269560783156 Unit:machined-test-register-with-network-segaprxj.service]
    dbus_test.go:132: machine machined-test-create-gvuxnxby properties: map[Class:container Id:[] Leader:4058828 Name:machined-test-create-gvuxnxby NetworkInterfaces:[] RootDirectory: Service:go-systemd State:running Timestamp:1729268546551546 TimestampMonotonic:269560785096 Unit:machine-machined\x2dtest\x2dcreate\x2dgvuxnxby.scope]
    dbus_test.go:132: machine machined-test-create-with-network-uhlgksmc properties: map[Class:container Id:[] Leader:4058831 Name:machined-test-create-with-network-uhlgksmc NetworkInterfaces:[] RootDirectory: Service:go-systemd State:running Timestamp:1729268546556992 TimestampMonotonic:269560790542 Unit:machine-machined\x2dtest\x2dcreate\x2dwith\x2dnetwork\x2duhlgksmc.scope]
--- PASS: TestMachine (0.10s)
=== RUN   TestImages
--- PASS: TestImages (0.00s)
PASS
ok      github.com/coreos/go-systemd/v22/machine1       0.106s
```

I will amend this, because I still think it's handy to have the check below. does it work?

Reply:
done